x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -74,7 +74,7 @@ export interface ResolverNodeStats {
 /**
  * A child node can also have additional children so we need to provide a pagination cursor.
  */
-export interface ChildNode extends LifecycleNode {
+export interface ResolverChildNode extends ResolverLifecycleNode {
```

Comment:
:grey_question: I've been trying to stop saying "Resolver" as much in favor of "GTV", I know they are changing the name for this officially, so it might be worth asking when we stop calling it that in our code? I'm not suggesting "right here, right now" just want to flag the issue.

Reply:
Few thoughts here:

1. I think that naming a project too early can be dangerous. If the scope and definition of a project is complete (won't change ever again), then naming it in a way that describes what it does is fine. But if you rename something like Resolver to 'Graphical Timeline', then people start thinking 'oh we should do A because this is a graphical timeline', or 'we shouldn't do b, because that doesn't fit the description of a graphical timeline.' Resolver isn't mature and its definition is going to be changing every release.
2. I'm hoping that this project isn't always specific to Security Solution. 'Graphical Timeline' is a branding term for a specific feature in Security Solution. That feature uses Resolver, but Resolver might be used elsewhere for other things. In other words, I'm thinking of Resolver as the mapping/graphing technology. 
3. If the term Graphical Timeline sticks around long enough that there becomes confusion, then we should reconsider the name of the project. I'm in a meeting right now and I've heard 'Resolver' said many times.
4. Graphical Timeline is an awkward name for a code project. Having a generic and vague name might be good for branding, but it would probably be confusing in our code base. For one thing, Resolver is entirely unrelated to Timeline, which is also a project in Security Solution.